docs/sdn_controller.md

```diff
@@ -26,11 +24,14 @@ In the network creation view:
 - Create the network
 - Have fun! ☺
 
-**_NB:_**
+![](./assets/sdn-controller.png)
+
+:::tip
 
 - All hosts in a private network must be able to reach the other hosts' management interface.
   > The term ‘management interface’ is used to indicate the IP-enabled NIC that carries the management traffic.
 - Only 1 encrypted GRE network and 1 encrypted VxLAN network per pool can exist at a time due to Open vSwitch limitation.
+  :::
```

Comment:
Note that this is non-standard syntax and it's formatting may be broken in the future by more standard tools we are using (eg Prettier).

Reply:
Ok